kolibri/plugins/learn/assets/test/views/content-card-group-carousel.spec.js

```diff
@@ -152,15 +155,29 @@ describe('ContentCardGroupCarousel', function() {
         });
 
         describe('LTR Languages', function() {
-          it('displays the first content item in the leftmost card');
+          it('displays the first content item in the leftmost card', function() {
+            return Vue.nextTick().then(()=>{
+              const firstCard = this.wrapper.first(card);
+              assert.ok(firstCard.hasStyle('left', '0'), 'CSS property "left" is not set to 0');
+            });
+          });
           describe('controls', function() {
-            // going to need to mount to dom for this
             it('contains a single button pointing right on the right');
           });
         });
 
         describe('RTL Languages', function() {
-          it('displays the first content item in the rightmost card');
+          it('displays the first content item in the rightmost card', function() {
+            const wrapper = this.wrapper;
+            const carousel = wrapper.instance();
+
+            // NOTE this lets the test pass... No idea why.
```

Comment:
@rtibbles even when returning the promise from `nextTick` directly, the tests passes. It's not until I `.then` it. :man_shrugging:

Reply:
oh and this seems to be happening whether I use `Vue` or `wrapper.instance()` (which is supposed to be `wrapper.vm` with a check confirming it's a component)

Reply:
Also, I know this test is actually failing because `isRtl` isn't set. The fact that this was passing it was alerted me to the false positives

Reply:
Yes, you should be `then`ing the promise from `Vue.nextTick` because you are waiting for the next tick, and then making the assertion. What you have now is the correct way of structuring it.

Reply:
I thought nextTick itself returns a promise, though, and the callback you're passing it doesn't happen until the nextTick?

Reply:
Ah yes, nextTick does support just passing a callback directly.